### DISCUSSION TITLE
Why a screenshot every time I quit?

### DISCUSSION THREAD

**Discussion by apprenti-sorcier** [opengisch/QField#5155] [Q&A]
Every time I quit my project, QField creates a screen capture as a png in the project folder. Why and is there a way to stop this useless action?
Thanks in advance.

**Comment by nirvn** [comment]
@apprenti-sorcier , the capture is to provide a project preview image attached to the recent projects'  list when launching QField.

**Comment by apprenti-sorcier** [comment]
Thanks for the explanation. This activity is useless in my case; not only I use just one project that loads automatically; but also it creates a file that is synchronised on my computer and it has nothing to do there. My QGIS project has a lot of data and QField takes a while to exit, therefore it’s not useful to make it longer with useless operations. I kept deleting the file every time and it still comes back. Please make this “feature” optional.
